index.html

```diff
@@ -153,7 +153,7 @@ <h1>Pull Requests <small>RIOT-OS/RIOT</small></h1>
 
   function connect_status() {
       // setup websocket with callbacks
-      var ws = new WebSocket('wss://ci.riot-labs.de/devel/status');
+      var ws = new WebSocket('wss://ci.riot-os.org/devel/status');
```

Comment:
Isn't the path `/status`, not `/devel/status` (same for `/api/pull_requests` above, but that seems to work even with `/devel`).